support/overlay/overlay.nix

```diff
@@ -83,7 +90,11 @@ in
     };
 
     amlogicGXL = aarch64.callPackage ../builders/amlogic-gxl {
-      gxlimg = final.Tow-Boot.gxlimg;
+      inherit (final.Tow-Boot) gxlimg;
```

Comment:
Please limit unrelated fixes and changes to specific discrete commits. The fixes like this one *are welcome*.

Otherwise it gets harder to `git blame` and sometimes even `git bisect` through unrelated changes.